src/EditorFeatures/TestUtilities/NavigateTo/AbstractNavigateToTests.cs

```diff
@@ -53,29 +60,43 @@ public abstract class AbstractNavigateToTests
         internal readonly static PatternMatch s_emptyCamelCaseNonContiguousSubstringPatternMatch_NotCaseSensitive = new PatternMatch(PatternMatchKind.CamelCaseNonContiguousSubstring, true, false, ImmutableArray<Span>.Empty);
         internal readonly static PatternMatch s_emptyFuzzyPatternMatch_NotCaseSensitive = new PatternMatch(PatternMatchKind.Fuzzy, true, false, ImmutableArray<Span>.Empty);
 
+        public static IEnumerable<object[]> TrackingServiceTypes
```

Comment:
@sharwell can we have some doc or sample on how we are supposed to do this? I dont think people in @dotnet/roslyn-ide knows all these magic since there are some new concepts to create tests.

basically, test now seem to require more knowledge on MEF for people to create unit tests.

Reply:
➡️ #28945